net/librdkafka/Portfile

```diff
@@ -13,10 +13,11 @@ description         The Apache Kafka C/C++ library
 long_description    Full Apache Kafka protocol support, including producer and consumer
 homepage            https://github.com/edenhill/librdkafka
 
-checksums           sha256  b0d16205b21e37280f3110db235988a1656f010c8dbc6700d86577fa435bd96c \
-                    rmd160  69f3eb84d52c4a5dd5dabb061d219c1022a24461 \
-                    size    2511625
+checksums           sha256 d6e553b1216b76de9e3a3c4b9c53fbecd87a5220aa5db2f3a8333d36e29e464c \
+                    rmd160 99496ebe9eefa3d197bdd893baca273ad142b758 
```

Comment:
Please retain the `size` parameter.

Reply:
ok, but it rather `add` then `retain`

Reply:
301a6a64ec5df74fcce5124187413c25fbf6b261

Reply:
I'm not sure what you mean, but that is what was requested.